src/tools/fuse/README.md

```diff
@@ -0,0 +1,189 @@
+# README.md
+
+Mount [elektra](https://www.libelektra.org)'s KDB as a [fuse(py)](https://github.com/fusepy/fusepy) filesystem.
+
+This tool enables the inspection and modification of the KDB, in the form of a classical filesystem, as it is observed by a given running process.
+In the simplest case, an Elektra key appears as a file with the key name as file path and with the key value as file content.
+
+This enables users to interact with Elektra
+- using standard unix tooling, making Elektra more accessible
+- with standard system/library procedures
+- without the need to learn other tooling of libelektra
+- with additional features: e.g. the ability to effortlessly write binary keys
+
+## Installation and Quickstart
+
+### Docker
+
+The package comes with a preconfigured docker environment for expirimentation/debugging/developement.
+
+Ensure docker is installed and run
+```sh
+cd src/tools/fuse/docker
+./run_new_container.sh
+```
+This (re)creates the container if necessary and starts an interactive session with keys for testing already created (see `docker/create_keys.sh`).
+The filesystem is mounted below `~/mount`.
+For debugging purposes, a logfile is written to `~/nohup.out` inside the container. 
+
+### Native
+
+System requirements:
+- Elektra of version at least 0.9.5 with the python binding installed (for example, `libelektra5-all` satisfies this)
+- `fuse` (for example via the debian package `fuse`) 
+- `python3 >= 3.8`, `pip3`
+
+Required python3 packages (will be installed by `pip3` automatically if not present):
+- `fusepy>=3.0.1`
+- `psutil>=5.8.0`
+
+Build the `elektra_fuse` python3-wheel and install it using `pip3` with:
+```
+./build.sh && sudo ./install.sh
```

Comment:
This should be integrated in a `make install`.

Reply:
Migrated to a makefile-workflow as of e4d3715c8485d00822d54e1dc5767b59826f39a9.